sdcm/sct_config.py

```diff
@@ -1312,7 +1312,8 @@ class SCTConfiguration(dict):
              help="""Use legacy cluster initialization with autobootsrap disabled and parallel node setup"""),
         dict(name="availability_zone", env="SCT_AVAILABILITY_ZONE",
              type=str,
-             help="Availability zone to use. Same for multi-region scenario."),
+             help="""Availability zone to use. Specify multiple (comma separated) to deploy resources to multi az (works on AWS).
```

Comment:
From the pipeline changes I would expect it to work also in Azure.
So, is it correct that only AWS really works now?

Reply:
Only AWS works now. Pipeline just disables `provision step` when multi-az is used.
In case Azure/GCE was used I expect it to fail at next stage.